doc/article/en-US/bundling-your-app-for-deploy.md

```diff
@@ -60,7 +60,7 @@ Now, let's create a `bundle.js` file in `build/tasks/bundle.js` as follows:
           includes: [
             '[*.js]',
             '*.html!text',
-            '*.css!text',        
```

Comment:
Fair enough. Though I think the one addition to a missing comma in this pr is still legitimate. I will can revert the comma removals if that is the desired syntax.